src/TouristInfoCards.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  Button
+} from "reactstrap";
+
+const TouristInfoCards = props => {
+  return (
+    <div className="col-12 d-lg-flex mx-auto">
+      <Card className="card bg-dark text-white flex-sm-column col-12 ml-auto mr-auto ml-md-4  mr-lg-2 col-md-10 mx-auto mx-md-auto col-lg-4 mt-md-5 mt-4 pr-2">
```

Comment:
This is looking great, but you're having to repeat a lot of the same information for all three cards here (e.g. you're saying that they contain a `CardImg` with a `top` property three times). Can you try extracting a component so that you can avoid some of that repetition?